content/en/docs/components/model-registry/getting-started.md

```diff
@@ -0,0 +1,170 @@
++++
+title = "Getting started"
+description = "Getting started with Model Registry using examples"
+weight = 30
+
++++
+
+This guide shows how to get started with Model Registry and run a few examples using the
+command line or Python clients.
+
+At this time, there is not yet an UI for Model Registry; therefore, this documentation focus on backend services and APIs.
+
+For an overview of the logical model of model registry, check the
+[Model Registry logical model](https://github.com/kubeflow/model-registry/blob/main/docs/logical_model.md).
+The logical model is exposed via the Model Registry [REST API](https://editor.swagger.io/?url=https://raw.githubusercontent.com/kubeflow/model-registry/main/api/openapi/model-registry.yaml).
+
+## Prerequisites
+
+To follow along the examples in this guide, you will need a Kubeflow installation and the Model Registry installed:
+
+- Kubeflow [installed](docs/started/installing-kubeflow/)
+- Model Registry [installed](/docs/components/model-registry/installation/)
+- Python >= 3.9, < 3.11
+
+## Example: track Model Artifacts from a Notebook
+
+This section details a step by step example on using Model Registry from a Notebook, installing and creating a client instance, indexing metadata, and retrieving metadata.
+
+### Install Model Registry Python client
+
+You can install the Model Registry python client in a Notebook, for instance with:
+
+```
+!pip install model-registry
+```
+
+Note: depending on your Python and Notebook environment, you might need to fine-tune the dependencies of: `ml-metadata`, `protobuf`, `grpcio`, or `tensorflow` if used.
+
+You can now create a client instance pointing to your deployed Model Registry from the previous steps.
+
+```python
+from model_registry import ModelRegistry
+
+registry = ModelRegistry(server_address="model-registry-service.kubeflow.svc.cluster.local", port=9090, author="your name")
+```
+
+You now have a Model Registry client instance: `registry`.
+
+### Register a Model Artifact metadata
+
+You can use the `register_model` method to index a model artifacts and its metadata, for instance:
+
+```python
+registeredmodel_name = "mnist"
+version_name = "v0.1"
+rm = registry.register_model(registeredmodel_name,
+                                "https://github.com/tarilabs/demo20231212/raw/main/v1.nb20231206162408/mnist.onnx",
+                                model_format_name="onnx",
+                                model_format_version="1",
+                                version=version_name,
+                                description="lorem ipsum mnist",
+                                metadata={
+                                    "accuracy": 3.14,
+                                    "license": "apache-2.0",
+                                }
+                                )
+```
+
+You can reference the pydoc documentation for additional information on indexing metadata on Model Registry, using the Model Registry Python client.
```

Comment:
suggested change - in the the Model Registry.     Note - this sentence has several prepositional phrases and it might be re-written.

Reply:
Revised as:

> For more information on indexing metadata in the Model Registry, refer to the pydoc documentation of the Model Registry Python client.

with https://github.com/kubeflow/website/pull/3698/commits/01490e130f4769786ba9bb86c5bebafa3ac1da14

thank you.